pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1881,6 +1908,9 @@ class Build {
                 // Run Smoke Tests and AQA Tests
                 if (enableTests) {
                     try {
+                        // Compare reproducible build, using same build pipeline with enableTests = false to avoid recursive building
+                        compareReproducibleBuild()
```

Comment:
Do we want this behind a enableReproducibleCheck flag?

Reply:
During the initial step the reproducible comparison is limited to nightly jdk17, linux-x64, temurin build only https://github.com/adoptium/ci-jenkins-pipelines/pull/714/files#diff-b4bb07e2f448f9ad2986d69423ad0b704dcdc4d217d947f465e6ace3fde7a4b3R594. Going forward might need to a flag setting per jdk version and platform.

Reply:
Updated. https://github.com/adoptium/ci-jenkins-pipelines/pull/714/commits/90635ee37a1afbd1737da81f036b6869b0f1ac4e